src/main/java/com/splunk/ibm/mq/opentelemetry/Main.java

```diff
@@ -45,6 +45,13 @@ public static void main(String[] args) throws Exception {
       System.exit(1);
     }
 
+    try {
+      Main.class.getClassLoader().loadClass("com.ibm.mq.headers.MQDataException");
+    } catch (ClassNotFoundException e) {
+      System.err.println("IBM MQ jar is missing from classpath.");
+      System.exit(1);
+    }
```

Comment:
This shouldn't be necessary -- the error will show itself as soon as that class is referenced. Why do this? Why not check every jar?

Reply:
For me it only showed later in the execution, and was swallowed.

Reply:
Well, I don't feel too strongly about this, but it still feels unnecessary. I think a better approach might be to locate where the swallow happened and wrap with an uncaught handler that logs it. I'll approve and leave that to you.

Reply:
You are working on that part. The exception swallowing happens while the task executes under WMQMonitor.doRun, when executed by the appd machine agent task execution framework.

Given we will never ship with IBM libs bundled, this is bound to happen often that customers will run the executable and not find that the issue is that the IBM jar is not present.